### ISSUE TITLE
How to build NNPACK for Android

### ISSUE THREAD

**Issue by waynehong666** [Maratyszcza/NNPACK#127]
There is no jni folder in NNPACK. How should I build NNPACK for Android.

**Comment by Maratyszcza** [comment]
NNPACK switched to CMake for most builds. To build it with the new build system:
```bash
cd NNPACK/build
cmake -DANDROID_TOOLCHAIN_FILE=$ANDROID_NDK/build/cmake/android.cmake.toolchain -D ANDROID_TOOLCHAIN=gcc ..
make
```
For additional options (e.g. how to build for a different Android ABI), see [Android NDK CMake guide](https://developer.android.com/ndk/guides/cmake.html)

**Comment by waynehong666** [comment]
When building the tests, I got the following error:
error: undefined reference to 'nnp_sgemm_only_6x8__aarch32_neon'

Are the tests working on Android devices?

**Comment by Maratyszcza** [comment]
@waynehong666: i reproduced the error, and fixed with 35864e570b112342ac0201909e8bfba47fc380f8
